microfrontends/SignDocuments/src/Component/SignYourselfPdf.js

```diff
@@ -1036,7 +1048,10 @@ function SignYourSelf() {
           >
             {/* this modal is used show this document is already sign */}
 
-            <Modal show={showAlreadySignDoc.status}>
+            <Modal 
+              show={showAlreadySignDoc.status}
+              onShow={() => modalAlign()}
+            >
               <ModalHeader style={{ background: themeColor() }}>
```

Comment:
There should be background color rgba(0, 0, 0, 0.75) as mentioned in #168  issue .

Reply:
Hello @raktima-opensignlabs background backdrop added for modal.
![image](https://github.com/OpenSignLabs/OpenSign/assets/33950743/93eed5e9-715b-48ae-9e27-7e7ca7852e50)